### PR TITLE
Add google analytics

### PR DESCRIPTION
**Pull request by mailtime-wing**

### CODE REVIEW

File: src/components/Footer/UpperSection/index.js [hunk 1]
```diff
@@ -31,19 +32,19 @@ import weiboIcon from "@/images/btn-weibo.svg"
...
 )
 
 const OtherPages = props => (
   <p css={theme => [fontStyle(theme), otherPages]} {...props} />
 )
 
-const Page = ({ children, ...props }) => (
-  <Link css={pageAnchor} {...props}>
+const Page = ({ children, to, ...props }) => (
+  <OutboundLink css={pageAnchor} href={to} {...props}>
```
Comment (chochihim) [inline src/components/Footer/UpperSection/index.js]: This is not outbound link. This is internal pages.

File: src/components/Header/index.js [hunk 2]
```diff
@@ -51,14 +51,14 @@ const Header = ({ headerMode }) => {
       }
     >
       <div css={content}>
-        <Link to="/">
+        <OutboundLink href="/">
```
Comment (chochihim) [inline src/components/Header/index.js]: This is not outbound link

File: gatsby-config.js [hunk 3]
```diff
@@ -5,6 +5,33 @@ module.exports = {
...
+        // Enables Google Optimize using your container Id
+        optimizeId: "YOUR_GOOGLE_OPTIMIZE_TRACKING_ID",
+        // Enables Google Optimize Experiment ID
+        experimentId: "YOUR_GOOGLE_EXPERIMENT_ID",
+        // Set Variation ID. 0 for original 1,2,3....
+        variationId: "YOUR_GOOGLE_OPTIMIZE_VARIATION_ID",
+        // Any additional optional fields
+        sampleRate: 5,
+        siteSpeedSampleRate: 10,
+        cookieDomain: "example.com",
```
Comment (chochihim) [inline gatsby-config.js]: Please check which option should be specified.

File: gatsby-config.js [hunk 4]
```diff
@@ -5,6 +5,33 @@ module.exports = {
     author: `@gatsbyjs`,
   },
   plugins: [
+    {
+      resolve: `gatsby-plugin-google-analytics`,
+      options: {
+        // The property ID; the tracking code won't be generated without it
+        trackingId: process.env.GOOGLE_ANALYTICS_TRACKING_ID,
```
Comment (chochihim) [inline gatsby-config.js]: Do not use env at this time. Hardcode first.

Comment (mailtime-wing) [reply gatsby-config.js]: done.

File: gatsby-config.js (hunk 3 above)
Comment (mailtime-wing) [reply gatsby-config.js]: Removed optional and unused fields in gatsby-config.js

File: src/components/Header/index.js (hunk 2 above)
Comment (mailtime-wing) [reply src/components/Header/index.js]: fixed.

File: src/components/Footer/UpperSection/index.js (hunk 1 above)
Comment (mailtime-wing) [reply src/components/Footer/UpperSection/index.js]: fixed.

File: gatsby-config.js [hunk 5]
```diff
@@ -5,6 +5,17 @@ module.exports = {
...
   plugins: [
+    {
+      resolve: `gatsby-plugin-google-analytics`,
+      options: {
+        // The property ID; the tracking code won't be generated without it
+        trackingId: "UA-138954130-2",
+        // Defines where to place the tracking script - `true` in the head and `false` in the body
+        head: true,
+        // Delays sending pageview hits on route update (in milliseconds)
+        pageTransitionDelay: 0,
```
Comment (chochihim) [inline gatsby-config.js]: I think these two options are also not required. Just let the plugin uses it own default value

**Pull request merged**